.github/workflows/call-calc-coverage.yml

```diff
@@ -4,15 +4,14 @@ name: call-calc_coverage
 # The default is to run the workflow on every push or pull request to every branch.
 on:
   workflow_dispatch:
-    paths-ignore:
-      - .devcontainer
-      - .github
-      - 'LICENSE'
-      - 'README.md'
-      - 'CONTRIBUTING.md'
-      - 'Rbuildignore'
-      - '.gitignore'
-      - 'man'
+  push:
+    branches:
+      - main
```

Comment:
Do we need to have push to main because there should only be PR to main? So, I guess I am thinking that the tests wouldn't take as long if we didn't double up on a PR and a push to main by just doing it on the PR. I am fine with leaving it though.

Reply:
Good point. It should only be a PR to main. I’ve made the changes, and @k-doering-NOAA, some changes from PR #720 might also need to be updated.